content/js/users.md

```diff
@@ -140,6 +140,11 @@ While each of the sign-up functions is resolved with a user object,
 in most cases you need not deal with it because
 you can access the currently logged-in user using `skygear.currentUser`.
 
+If your app does not require the user to create an account before using your
+app, you can sign up the user anonymously. An user that is signed up
+anonymously does not have a username, email, nor a password. You can add a
+username, email and/or password to a user after signing up.
```

Comment:
Does it mean that a new user will be created for every anonymous signup? I am not sure about this feature - are there any implications on how to handle the anonymous user?

- each of these users still has their own user IDs
- can an anonymous user create records?
- does an anonymous user have a default role?